### PR TITLE
Editor fix

### PR DESCRIPTION
**Pull request by everett980**
# this needs to merge before anything else!

### CODE REVIEW

File: game/js/states/load.js [hunk 1]
```diff
@@ -83,13 +83,11 @@ function initLoadState() {
...
-          skyColor: '#FFBB22',
-          startGirders: 12,
-          objects: data[1]
-        };
-
+				game.level = {
+					skyColor: data[1].skyColor,
+					startGirders: data[1].girdersAllowed,
+					objects: data[1].levelArr
+				};
```
Comment (LanaCoyote) [inline game/js/states/load.js:90]: :+1:

**Comment by LanaCoyote** [comment]
I approve

**Comment by LanaCoyote** [comment]
I rescind my approval, either lock mongoose to an older version or use the new promisifed database connection.

**Comment by everett980** [comment]
new promisified database connection isn't working. the revertion i made works for me and yustynn and brian dunn after the changes broke the connection.

**Comment by everett980** [comment]
this merge would close #62 and #71

**Comment by LanaCoyote** [comment]
Let me push up a commit to make it work for everyone

**Comment by LanaCoyote** [comment]
Let me know if that doesn't work for you

**Comment by everett980** [comment]
... Uhoh. Got error listen EADDRINUSE :::35729 ...
Error: listen EADDRINUSE :::35729
    at Object.exports._errnoException (util.js:874:11)
    at exports._exceptionWithHostPort (util.js:897:20)
    at Server._listen2 (net.js:1234:14)
    at listen (net.js:1270:10)
    at Server.listen (net.js:1366:5)
    at Server.listen (/Users/everett980/GitHub/girderGusMaker/node_modules/gulp-livereload/node_modules/mini-lr/lib/server.js:162:15)
    at Function.exports.listen (/Users/everett980/GitHub/girderGusMaker/node_modules/gulp-livereload/index.js:95:18)
    at Gulp.<anonymous> (/Users/everett980/GitHub/girderGusMaker/gulpfile.js:263:14)
    at module.exports (/Users/everett980/GitHub/girderGusMaker/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/everett980/GitHub/girderGusMaker/node_modules/gulp/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Users/everett980/GitHub/girderGusMaker/node_modules/gulp/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/Users/everett980/GitHub/girderGusMaker/node_modules/gulp/node_modules/orchestrator/index.js:134:8)
    at /usr/local/lib/node_modules/gulp/bin/gulp.js:129:20
    at doNTCallback0 (node.js:417:9)
    at process._tickCallback (node.js:346:13)
    at Function.Module.runMain (module.js:469:11)

You already have a server listening on 35729
You should stop it and try again.

**Comment by LanaCoyote** [comment]
Restart your gulp process

**Comment by everett980** [comment]
i had six instances of node running within the same tab of iterm, not sure how i accomplished that, but it happened.
restarted iterm, gulp & npm start -> works

**Comment by LanaCoyote** [comment]
Ok, I'm good with this PR. Unless Yustynn has concerns you can merge whenever

**Comment by Yustynn** [comment]
I'm fine. Merging now.

**Pull request merged**